### DISCUSSION TITLE
kubernetes-handbook/concepts/spire/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#31] [Announcements]
# kubernetes-handbook/concepts/spire/

这篇文章将向你介绍 SPIRE 的架构、基本概念及原理。

https://lib.jimmysong.io/kubernetes-handbook/concepts/spire/

**Comment by rootsongjc** [comment]
这本书迁移自我的 [Kubernetes Handbook](https://jimmysong.io/kubernetes-handbook/)，去掉了 Gitbook 中与 Kubernetes 无关的内容，删除了一些过时的内容。
